caniusepython3/test/test_cli.py

```diff
@@ -159,22 +159,37 @@ def test_message_singular(self):
         self.assertEqual(2, len(messages))
         want = 'You need 1 project to transition to Python 3.'
         self.assertEqual(messages[0], want)
-        want = ('Of that 1 project, 1 has no direct dependencies blocking '
+        want = ('Of that 1 project, 1 has no direct dependencies known to block '
                 'its transition:')
         self.assertEqual(messages[1], want)
 
+    def test_message_with_unknowns(self):
+        unknowns = [['A']]
+        messages = ciu_main.message([])
+        expected = ['You have no projects known to block you from using Python 3!']
+        self.assertEqual(expected, messages)
+
+    def test_output_unknowns(self):
+        unknowns = ['A']
+        outputs = ciu_main.output_not_on_pypi(unknowns)
+        self.assertEqual(2, len(outputs))
+        want = 'The following 1 project could not be found on pypi.org:'
+        self.assertEqual(outputs[0], want)
```

Comment:
No need to be so strict on the output. Just check that the string appears _somewhere_ in the output.
```suggestion
        self.assertIn(want, outputs[0])
```
Same with the other output tests.